sphinx/texinputs/latexmkjarc

```diff
@@ -0,0 +1,7 @@
+$latex = 'platex --halt-on-error --interaction=nonstopmode -kanji=utf8 %O %S';
+$dvipdf = 'dvipdfmx %O -o %D %S';
+$makeindex = 'mendex -U -f -d %B.dic -s python.ist %S || echo "mendex exited with error code $? (ignoring)" && : >> %D';
```

Comment:
My proposal is ...
* `Makefile` and `latexmkrc` is used for Unix like OS
* `make.bat` and `latexmkrc.bat` is used for Windows platform

In windows, `make.bat` on current directory is preferentially used than `make.exe`. So providing `make.bat` helps to switch latexmkrc files by platform.